.clang-format

```diff
@@ -130,8 +130,10 @@ Standard:        Latest
 StatementMacros:
   - Q_UNUSED
   - QT_REQUIRE_VERSION
-TabWidth:        8
-UseCRLF:         false
-UseTab:          Never
+TabWidth:           8
+UseCRLF:            false
+UseTab:             Never
+InsertBraces:       true
+InsertNewlineAtEOF: true
```

Comment:
Many linux tools get mad if there isn't a single newline at end of file. This helps those tools not get grumpy.

It only works with clang 16 or newer, so the majority of developers probably won't have it, but that's ok, it'll be ignored by versions of clang-format that are too old.

Reply:
Ugh, nevermind, it breaks the CI.